sycl/doc/extensions/deprecated/sycl_ext_oneapi_discard_queue_events.asciidoc

```diff
@@ -0,0 +1,211 @@
+= sycl_ext_oneapi_discard_queue_events
+:source-highlighter: coderay
+:coderay-linenums-mode: table
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+
+:blank: pass:[ +]
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+// This is necessary for asciidoc, but not for asciidoctor
+:cpp: C++
+
+== Introduction
+
+IMPORTANT: This specification is a draft.
+
+NOTE: Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are
+trademarks of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc.
+used by permission by Khronos.
+
+This document describes an extension that introduces a `discard_events` property for
+SYCL queues. This property enables developers to inform a SYCL implementation that
+the events returned from queue operations will not be used.
+
+== Notice
+
+Copyright (c) 2021 Intel Corporation. All rights reserved.
+
+== Status
+
+This extension has been deprecated.  Although the interfaces defined in this
+specification are still supported in {dpcpp}, we expect that they will be
+removed in an upcoming {dpcpp} release.  The optimizations enabled by these
+interfaces have already been disabled in the compiler.  The functionality of
+this extension has been replaced by the sycl_ext_oneapi_enqueue_functions
+extension: see link:../experimental/sycl_ext_oneapi_enqueue_functions.asciidoc[here].
+*Shipping software products should stop using APIs defined in this
+specification and use this alternative instead.*
+

```

Comment:
I think you could add some words here noting that this extension no longer provides any benefit, and direct people to the enqueue function extension as an alternative.

Reply:
I've added the sentence "This extension no longer provides any benefit" before the existing explanation and sentence pointing to the enqueue_functions extension. I've also copied this to the doc in the supported folder. Let me know if you think I should change it any further.
Thanks